crates/uv-platform-tags/src/platform_tag.rs

```diff
@@ -262,6 +265,7 @@ impl std::fmt::Display for PlatformTag {
             Self::Haiku { release_arch } => write!(f, "haiku_{release_arch}"),
             Self::Illumos { release_arch } => write!(f, "illumos_{release_arch}"),
             Self::Solaris { release_arch } => write!(f, "solaris_{release_arch}_64bit"),
+            Self::Pyodide { major, minor } => write!(f, "pyodide_{major}_{minor}_wasm32"),
```

Comment:
@hoodmane -- Should this not be `emscripten_3_1_58_wasm32`, for `pyodide_2024_0`?

Reply:
I put up a PR here, I may be mistaken though: https://github.com/astral-sh/uv/pull/14226

Reply:
Ah okay, I'm seeing that the wheels in `pyodide config get package_index` _do_ use this pyodide tag. Should we support both?